ocp-templates/templates/cd-jenkins-webhook-proxy.yml

```diff
@@ -114,5 +116,7 @@ parameters:
   value: cd
 - name: REPO_BASE
   value: REPO_BASE
+- name: PROTECTED_BRANCHES
+  value: "master,develop,production,staging,release/"
```

Comment:
Is the '/' only at release intended?

I am seeing this as default value https://github.com/opendevstack/ods-core/blob/aa5f1c48570554822e44096e90fa8d0ee4bb04d1/jenkins/webhook-proxy/main.go#L29

Reply:
Nevermind, saw a subsequent PR now, where this matches.